### PR TITLE
Add Dialog.enable annotation to parameter IC in C_Capacitor, C_SEMI and L_Inductor

### PR DESCRIPTION
**Pull request by beutlich**

### CODE REVIEW

**Comment by beutlich** [comment]
This needs to be modified after merging #3235.

File: Modelica/Electrical/Spice3.mo [hunk 1]
```diff
@@ -2288,7 +2288,7 @@ The Resistance <em>R</em> is allowed to be positive, zero, or negative.
       parameter SI.Capacitance C(start=0) "Capacitance";
       parameter SI.Voltage IC=0 "Initial value";
       parameter Boolean UIC=false
-        "Use initial conditions: true, if initial condition is used";
+        "Use initial conditions: true, if initial condition is used" annotation(Dialog(enable=false));
```
Comment (dietmarw) [inline Modelica/Electrical/Spice3.mo]: How will you be able to set  UIC?

File: Modelica/Electrical/Spice3.mo [hunk 2]
```diff
@@ -2288,7 +2288,7 @@ The Resistance <em>R</em> is allowed to be positive, zero, or negative.
       parameter SI.Capacitance C(start=0) "Capacitance";
       parameter SI.Voltage IC=0 "Initial value";
```
Comment (christiankral) [inline Modelica/Electrical/Spice3.mo]:
```suggestion
       parameter SI.Voltage IC=0 "Initial value" annotation(Dialog(enable=UIC));
```

File: Modelica/Electrical/Spice3.mo (hunk 1 above)
Comment (christiankral) [reply Modelica/Electrical/Spice3.mo]:
```suggestion
        "Use initial conditions: true, if initial condition is used";
```

**Pull request merged**